.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish to PyPI
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
There may be a better trigger for this workflow based on the preferred release process.

Reply:
@PSalant726 check fidesctl we do it in a smarter way for the docs -> https://github.com/ethyca/fides/blob/main/.github/workflows/publish_docs.yaml#L7

Reply:
Updated. @seanpreston does this now align with the current release process?